Resources/Server Info/RP_Rules.txt

```diff
@@ -0,0 +1,103 @@
+    - What is considered a reasonable reason to escalate conflict is listed below under ESCALATION RULES and is at the discretion of admins.
+    - Declaring a revolution or going on strike is an antagonist only activity. “Cargonia” and other departmental revolutions are prohibited unless admin permission is obtained.
+[color=#a4885c]5.[/color] Erotic Roleplay (ERP) is [color=#ff0000]not allowed under any circumstance[/color]. This includes any direct or implied sexual acts as well as explicit mentions of sexual behavior.
+[color=#a4885c]6.[/color] Harassment of other players for OOC/meta reasons is [color=#ff0000]not allowed[/color].
+	- Annoying somebody in a round for IC/RP reasons is fine. Doing it consistently across multiple rounds for OOC reasons (“I don’t like this player” AKA metagrudging) to the point where it becomes seriously problematic for the other player(s) is [color=#ff0000]not[/color].
+
+[color=#a4885c]7.[/color] Hate speech, slurs and bigotry are not allowed.
+	- Intentionally seeking to demean others due to their actual or perceived race, sex, gender, sexual orientation or the like is not tolerated.
+	- Words that are closely tied to real life slurs are [color=#ff0000]not allowed[/color].
+    - Species-ism (IC or OOC bigotry against other in-game species) is also NOT ALLOWED. Not even for roleplay or because it's your "character".
+
+[color=#a4885c]8.[/color] Offensive/bad/out-of-character character names and descriptions put you at the mercy of administrators, from receiving brain damage to becoming valid to straight up getting banned on the spot depending on how bad they are.
+    - Character names should be something that a member of that species would reasonably have.
+    - No famous real or fictional names of characters.
+
+[color=#a4885c]9.[/color] Knowingly using exploits, external programs, or other unintended game mechanics to gain in-game advantages or to lag/crash the server is [color=#ff0000]not allowed[/color].
+
+[color=#a4885c]10.[/color] Multi-keying/Alts on the same server is [color=#ff0000]not allowed[/color].
+	- Admins may ask players in possession of multiple alt accounts to choose one to play on and ban the other accounts.
+	- If you are caught knowingly using alt accounts you may have your main account banned as well.
+
+[color=#a4885c]11.[/color] Using information gained outside the game to your advantage is [color=#ff0000]not allowed[/color]. IC and OOC are to be kept separate at all times.
+	- Metacommunication, or communicating with other players via out of game methods (such as Discord) to gain an unfair advantage is also strictly [color=#ff0000]prohibited[/color].
+    - This is a roleplaying game, thus IC conflicts must only be resolved with IC (Local) chat. Utilizing voice chat, calling across the room, OOC, or other out-of-game communication methods to coordinate in-game actions is forbidden and will result in a [color=#ff0000]appeal-only permanent ban[/color] in most cases.
```

Comment:
```suggestion
    - This is a roleplaying game, thus IC conflicts must only be resolved with IC (Local) chat. Utilizing voice chat, OOC, or other out-of-game communication methods to coordinate in-game actions is forbidden and will result in a [color=#ff0000]appeal-only permanent ban[/color] in most cases.
```
Very weird example.

Reply:
its a weird example because it literally happened im fairly sure